packages/framework/src/Form/Admin/Category/CategoryFormType.php

```diff
@@ -257,4 +261,40 @@ private function getCategoryNameForPlaceholder(DomainConfig $domainConfig, ?Cate
 
         return $category === null ? '' : $category->getName($domainLocale);
     }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param \App\Model\Category\Category|null $category
```

Comment:
```suggestion
     * @param \Shopsys\FrameworkBundle\Model\Category\Category|null $category
```
App namespace must not be used in the framework